src/controllers/IframeController.tsx

```diff
@@ -33,7 +33,7 @@ export default function IframeController({ currentConfig }: { currentConfig: Web
   // navigation
   const currentStep = useCurrentStep();
   const navigate = useNavigate();
-  const computedTo = useNextStep();
+  const { nextStep: computedTo } = useNextStep();
```

Comment:
It will be cleaner and easier to debug this code if we just change the variable to `nextStep`]

```suggestion
  const { nextStep } = useNextStep();
```

Reply:
Actually, I'm not sure if we even need this. It's just used as a dependency of the useEffect below, but since we also have currentStep, it should be okay without it